src/ImpactEvaluator.sol

```diff
@@ -8,16 +8,17 @@ pragma solidity ^0.8.19;
 contract ImpactEvaluator is AccessControl, Nonces {
     struct Round {
         uint index;
+        uint end;
```

Comment:
I find it a  bit difficult to quickly understand what is the value type (unit?) stored in these two fields.

IIUC:
- `index` is the IE round index
- `end` is the last block number of this round

I am proposing to rename `end` to something like `lastBlockNumber` or `lastEpoch` to make it more clear what value is in this field.

Maybe we can rename `index` to `roundIndex` too, although it also feels a bit redundant for `currentRound.roundIndex` 🤷🏻

Reply:
I went with `endBlockNumber`, to both have a small change and match existing variable `block.number`. Or do you prefer `lastBlockNumber`?

With the `index`, I think `round.index` is self explanatory enough, and we're never using that `index` independently of the `round` struct

Reply:
`endBlockNumber` is good 👍🏻 Thanks!